backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -14,9 +14,16 @@ You back up applications by creating a `Backup` custom resource (CR). See xref:.
 
 For more information about CSI volume snapshots, see xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI volume snapshots].
 
-:FeatureName: The `CloudStorage` API for S3 storage
+:FeatureName: The `CloudStorage` API, which automates the creation of a bucket for object storage,
 include::snippets/technology-preview.adoc[]
 
+[NOTE]
+====
+The `CloudStorage` API supports manually creating a `BackupStorageLocation` object specifying an existing S3 bucket. The `CloudStorage` API that creates an S3 bucket automatically, currently is only enabled for AWS S3 storage.
+
+The `CloudStorage` API is a Technology Preview, if you use a `CloudStorageLocation` object and want OADP to use the `CloudStorage` API to automatically create an S3 bucket, for use as a `BackupStorageLocation` with GCP and Microsoft Azure storage.
```

Comment:
This sentence is also confusing, maybe because of the commas. Because of them, I'm not sure the direction that the sentence is trying to take. I think it might be...

```suggestion

The `CloudStorage` API is currently in Technology Preview. If you use a `CloudStorageLocation` object and want OADP to automatically create an S3 bucket for use as a `BackupStorageLocation` with GCP and Microsoft Azure storage, you can enable this feature.
```

But I'm not 100% sure if this is what you're aiming for.